examples/packet_forward_test.go

```diff
@@ -0,0 +1,191 @@
+package ibctest_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	transfertypes "github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
+	"github.com/strangelove-ventures/ibctest"
+	"github.com/strangelove-ventures/ibctest/ibc"
+	"github.com/strangelove-ventures/ibctest/test"
+	"github.com/strangelove-ventures/ibctest/testreporter"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+)
+
+func TestPacketForwardMiddleware(t *testing.T) {
+	t.Skip()
+
+	home := t.TempDir()
+	client, network := ibctest.DockerSetup(t)
+
+	rep := testreporter.NewNopReporter()
+	eRep := rep.RelayerExecReporter(t)
+
+	ctx := context.Background()
+
+	cf := ibctest.NewBuiltinChainFactory(zaptest.NewLogger(t), []*ibctest.ChainSpec{
+		{Name: "gaia", ChainName: "gaia-fork", Version: "bugfix-replace_default_transfer_with_router_module"},
+		{Name: "osmosis", ChainName: "osmosis", Version: "v7.3.0"},
+		{Name: "juno", ChainName: "juno", Version: "v6.0.0"},
+	})
+
+	chains, err := cf.Chains(t.Name())
+	require.NoError(t, err)
+
+	gaia, osmosis, juno := chains[0], chains[1], chains[2]
+
+	r := ibctest.NewBuiltinRelayerFactory(ibc.CosmosRly, zaptest.NewLogger(t)).Build(
+		t, client, network, home,
+	)
+
+	const pathOsmoHub = "osmohub"
+	const pathJunoHub = "junohub"
+
+	ic := ibctest.NewInterchain().
+		AddChain(osmosis).
+		AddChain(gaia).
+		AddChain(juno).
+		AddRelayer(r, "relayer").
+		AddLink(ibctest.InterchainLink{
+			Chain1:  osmosis,
+			Chain2:  gaia,
+			Relayer: r,
+			Path:    pathOsmoHub,
+		}).
+		AddLink(ibctest.InterchainLink{
+			Chain1:  gaia,
+			Chain2:  juno,
+			Relayer: r,
+			Path:    pathJunoHub,
+		})
+
+	require.NoError(t, ic.Build(ctx, eRep, ibctest.InterchainBuildOptions{
+		TestName:  t.Name(),
+		HomeDir:   home,
+		Client:    client,
+		NetworkID: network,
+
+		SkipPathCreation: false,
+	}))
+
+	const userFunds = int64(10_000_000_000)
+	users := ibctest.GetAndFundTestUsers(t, ctx, t.Name(), userFunds, osmosis, gaia, juno)
+
+	channels, err := r.GetChannels(ctx, eRep, gaia.Config().ChainID)
+	require.NoError(t, err)
+
+	// Start the relayer on both paths
+	err = r.StartRelayer(ctx, eRep, pathOsmoHub)
+	require.NoError(t, err)
+
+	err = r.StartRelayer(ctx, eRep, pathJunoHub)
+	require.NoError(t, err)
+
+	t.Cleanup(
+		func() {
+			err := r.StopRelayer(ctx, eRep)
```

Comment:
I think this will hit a relayer bug -- IIRC there is a `container` field on the relayer type that will be overwritten on the second call to `r.StartRelayer`.

This is fine for now but we should fix that separately.